packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -4,15 +4,16 @@ import {
   EventEmitter,
   Host,
   Prop,
+  State,
   h,
   Element,
 } from '@stencil/core';
 
 import {
   months,
   days,
-  isFullDate,
-  isMonthYearDate,
+  // isFullDate,
+  // isMonthYearDate,
```

Comment:
Did you mean to leave these commented out?

Reply:
For now, yes. I left a comment wondering about why they were used in the first place considering the rest of the boolean expression, so I'm not totally sure that removing them doesn't break things. If we decide that they don't need to be there then I'll actually remove the functions.